modules/apps/export-import/export-import-web/src/main/java/com/liferay/exportimport/web/internal/servlet/filter/MockServiceWorkerContextFilter.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.exportimport.web.internal.servlet.filter;
+
+import com.liferay.portal.kernel.feature.flag.FeatureFlagManagerUtil;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.servlet.BaseFilter;
+
+import jakarta.servlet.Filter;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Jaime León
+ */
+@Component(
+	property = {
+		"servlet-context-name=",
+		"servlet-filter-name=Mock Service Worker Context Filter",
+		"url-pattern=/o/exportimport-web/mockServiceWorker.js"
+	},
+	service = Filter.class
+)
+public class MockServiceWorkerContextFilter extends BaseFilter {
+
+	@Override
+	protected Log getLog() {
+		return _log;
+	}
+
+	@Override
+	protected void processFilter(
+			HttpServletRequest httpServletRequest,
+			HttpServletResponse httpServletResponse, FilterChain filterChain)
+		throws Exception {
+
+		if (FeatureFlagManagerUtil.isEnabled("LPD-11309")) {
```

Comment:
Should we pass the `companyId` to the `isEnabled` method? I think we can get it from the `User` of the `HttpServletRequest` or from the `Portal _portal`

Reply:
:heavy_check_mark: